Plugin/Block/Widget/Block.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Mageplaza
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Mageplaza.com license that is
+ * available through the world-wide-web at this URL:
+ * https://www.mageplaza.com/LICENSE.txt
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this extension to newer
+ * version in the future.
+ *
+ * @category    Mageplaza
+ * @package     Mageplaza_Productslider
+ * @copyright   Copyright (c) Mageplaza (https://www.mageplaza.com/)
+ * @license     https://www.mageplaza.com/LICENSE.txt
+ */
+
+namespace Mageplaza\Productslider\Plugin\Block\Widget;
+
+use Magento\Cms\Block\Widget\Block as WidgetBlock;
+use Mageplaza\Productslider\Helper\Data as HelperData;
+use Magento\Framework\App\RequestInterface;
+
+/**
+ * Class Block
+ * @package Mageplaza\Productslider\Plugin\Block\Widget
+ */
+class Block
+{
+    /**
+     * @var HelperData
+     */
+    protected $helperData;
+
+    /**
+     * @var RequestInterface
+     */
+    protected $request;
+
+    /**
+     * @param HelperData $helperData
+     * @param RequestInterface $request
+     */
+    public function __construct(
+        HelperData $helperData,
+        RequestInterface $request
+    ) {
+        $this->helperData = $helperData;
+        $this->request    = $request;
+    }
+
+    /**
+     * @param WidgetBlock $subject
+     * @param $result
+     *
+     * @return mixed|void
+     */
+    public function afterToHtml(WidgetBlock $subject, $result)
```

Comment:
This implementation of afterToHtml causes all other widget blocks on my site (not product sliders) to disappear.  I think the condition needs to be adjusted to only take effect on mageplaza product slider instances, but not sure how to do that.

Reply:
In the meantime, I've reverted to a previous version.